Added more songs in Openings & OST

saideepd opened the pull request:
Made some additions as per #195 
#### This PR is for [Re:Zero Kara Hajimeru Isekai Seikatsu, Bleach, Detective Conan, One Piece, Inazuma Eleven, Death Note].

- [x] This issue is for enhancement/bug fixing.
- [x] This issue is an addition to JSON database.
	- [x] Pull Request Title is of form `Added [Re:Zero Kara Hajimeru Isekai Seikatsu, Bleach, Detective Conan, One Piece, Inazuma Eleven, Death Note]`.
- [ ] This addition is already on ongaku.
	- [ ] This PR is an improvement to current addition.
- [x] I have checked my addition locally and it is working and won't break existing entries.
- [x] I have gone through [Contribution Guidelines](https://github.com/anshumanv/ongaku/blob/master/CONTRIBUTING.md).
- [ ] This pull request will be closed if I failed to update it even once in a week.

saideepd committed:
Added more songs in Openings & OST

saideepd committed:
Fixed OST

saideepd commented:
Added more Opening & OST, fixed some bugs.

saideepd commented:
Sorry for closing the pull request by mistake. My branch has passed all the tests. Please merge it with master branch.

anshumanv commented:
Thanks, @saideepd for the PR.
I'll merge once reviewed. 😄

anshumanv commented on js/data.json:
```diff
@@ -1280,6 +1292,30 @@ osts = [
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/3l41690wfwq3vvo/real%20bleach%20opening%201%20orange%20range%20asterisk.m4a?dl=0",
+		"name" : "Bleach OST 1 - Asterisk - Orange Range",
+		"img" : "https://i.ytimg.com/vi/LGAYnGcOd2o/maxresdefault.jpg"
```
Use some better image like an anime scene or something, also this is OP1 so please place this in openings section.

anshumanv commented on js/data.json:
```diff
@@ -520,6 +520,18 @@ openings = [
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/27m8fylvfa42gsa/Inazuma%20Eleven%20Opening%20Theme%20Song.m4a?dl=0 ",
+		"name" : "Inazuma Eleven OST - Opening Theme Song",
```
Isn't this OP ?

anshumanv commented on js/data.json:
```diff
@@ -1280,6 +1292,30 @@ osts = [
+	
+	{
+		"link" : "https://www.dl.dropboxusercontent.com/s/9sqnxnojdtrcjnm/Detective%20Conan%20Soundtrack%201.m4a?dl=0",
+		"name" : "Detective Conan OST 1",
```
Does this OST have a name? If so please add it

anshumanv reviewed (changes requested):
Please check comments

saideepd commented:
Sure. Thanks @anshumanv for the reply. Hope that my PR will get merged 😄

anshumanv commented:
I am seeing that you're using dropbox links, are you sure the links will stay in place?

saideepd commented:
Yes. I have hosted them on my personal Dropbox account. And I don't use Dropbox a lot, so I won't be deleting them.

anshumanv commented:
Alright then, I'll merge after you make requested changes. 😄

saideepd committed:
Made changes as Requested

Moved Inazuma from OP to OST, used better image for Bleach & moved it to OP, added OST name to Detective Conan

saideepd commented:
I have made changes as requested. Please review it now.

saideepd committed:
Moved Inazuma to OP from OST

anshumanv committed:
Update re:zero OP name

saideepd commented:
@anshumanv Thanks for merging the PR bro 👍

anshumanv commented:
Thanks for the PR @saideepd, you can continue to add more if you wish. 😄

saideepd commented:
Sure 👍 @anshumanv
